recipes/inference-server/meta.yaml

```
@@ -11,19 +11,18 @@ source:
 
 build:
   noarch: python
-  skip: true  # [py<38]
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
 requirements:
   host:
-    - python
+    - python >=3.9
     - setuptools >=45
     - wheel
     - setuptools-scm >=6.2
     - pip
   run:
-    - python
+    - python >=3.9
     - botocore
     - codetiming >=1.4,<2.dev0
     - importlib-metadata <4  # [py<38]
```

It is OK to install this in py>=38, the standard lib version will be used instead.

```suggestion
    - importlib-metadata <4 
```

---

Ah yes, missed that one. All fixed.